stone/target/python_type_mapping.py

```diff
@@ -0,0 +1,88 @@
+from typing import (
```

Comment:
Do you think it would be reasonable to say that all typing imports should be prefixed with `T`?

For example,
```
from typing import (
    TAny,
    TCallable,
    TDict,
)
```

Standardizing this will help reduce the name conflicts caused by typing a typing system :)